addons/website_blog/static/src/js/s_latest_posts_editor.js

```diff
@@ -32,14 +32,11 @@ sOptions.registry.js_get_posts_limit = sOptions.Class.extend({
      * @override
      */
     _setActive: function () {
-        var self = this;
         this._super.apply(this, arguments);
-        this.$('[data-posts-limit]').addBack('[data-posts-limit]')
-            .removeClass('active')
-            .filter(function () {
-                return (self.$target.data('postsLimit') || 3) == $(this).data('postsLimit');
-            })
-            .addClass('active');
+        var activeLimit = this.$target.data('postsLimit') || 3;
+
+        this.$el.find('[data-posts-limit]').addBack('[data-posts-limit]').removeClass('active');
+        this.$el.find('[data-posts-limit='+ activeLimit + ']').addClass('active');
```

Comment:
Well...  previous implementations it's faster but it doesn't work ^^, for what can I see It's broken since v12.
Anyway I think that we can remove `addBack`